splinter/browser.py

```diff
@@ -26,6 +26,8 @@
     'zope.testbrowser': None,
 }
 
+LOG = logging.getLogger(__name__)
```

Comment:
nit: can we call the variable `logger` instead of `LOG`?

Reply:
certainly